### ISSUE TITLE
Add carousel to Instagram widget

### ISSUE THREAD

**Issue by stasonua0** [Crocoblock/suggestions#412]
Often there is a need to display a photo from instagram carousel. Please add this opportunity in Jet Elements.

/*Оригинал*/
Часто возникает необходимость выводить фото из instagram каруселью. Добавьте пожалуйста такую возможность в Jet Elements

**Comment by lkrzyczkowski** [comment]
Yes this is good option to have carusel effect in instagram widget.

**Comment by thorsolutions** [comment]
Dear Crocoblock, Please concentrate on fixing your existing widgets before adding more features now. A lot of the widgets need work as you can see from the github.
